plugins/services/src/js/components/modals/NewCreateServiceModalForm.js

```diff
@@ -422,10 +451,38 @@ class NewCreateServiceModalForm extends Component {
     ];
   }
 
+  /**
+   * This function filters the error list in order to keep only the
+   * errors that should be displayed to the UI.
+   *
+   * @returns {Array} - Returns an array of errors that passed the filter
+   */
+  getUnmutedErrors() {
```

Comment:
Hmm.. this is very specific case to `NewCreateServiceModalForm`. Which part do you think might be good to be separated?